src/net/gateway-archive.ts

```diff
@@ -0,0 +1,151 @@
+import { FileApi } from "../api/file";
+import { GatewayArchiveError } from "../errors/gateway-archive";
+import { TextRecordKeys } from "../models/entity"
+import { VochainProcessStatus } from "../models/protobuf";
+import { getEnsPublicResolverByNetwork } from "../util/ens";
+import { ContentUri } from "../wrappers/content-uri"
+import { EthNetworkID, IGatewayClient, IGatewayDVoteClient } from "../common";
+
+export interface IArchiveResponseBody {
+    process?: {
+        [k: string]: any
```

Comment:
Isn't there a type definition for this? ProcessDetails? 🤔

Reply:
I think it's `ProcessSummary`

Reply:
In such case, let's better use it :)